listenbrainz_spark/recommendations/utils.py

```diff
@@ -9,3 +15,40 @@ def save_html(filename, context, template):
     with open(outputfile, 'w') as f:
         html = template_environment.get_template(template).render(context)
         f.write(html)
+
+def initialize_spark_session(app_name):
+    try:
+        listenbrainz_spark.init_spark_session(app_name)
+        return True
+    except AttributeError as err:
+        logging.error('Cannot initialize Spark session "{}": {} \n {}.'.format(app_name, type(err).__name__,
+            str(err)), exc_info=True)
+        return False
+    except Exception as err:
+        logging.error('An error occurred while initializing Spark session "{}": {} \n {}.'.format(app_name,
+            type(err)  .__name__,str(err)), exc_info=True)
+        return False
+
+def register_dataframe(df, table_name):
+    try:
```

Comment:
This function is useful for stats also, let's put it inside listenbrainz_spark/__init__.py or listenbrainz_spark/utils.py